ia/Makefile

```
@@ -1,26 +1,54 @@
+ensure-deps:
+	@if ! python3 -c "import coverage" 2>/dev/null; then \
+		echo "Installing coverage..."; \
+		pip install coverage; \
+	fi
+
+coverage-html: ensure-deps
+	@echo "Generating HTML coverage report..."
+	@python3 -m coverage html
+	@echo "HTML coverage report generated in htmlcov/ directory"
+
 clean:
```

checkmake (0.2.2)

[warning] Target body for "clean" exceeds allowed length of 5 (7).

(maxbodylength)